### DISCUSSION TITLE
Watch face not rotating 180 degrees on Wear Os for left handed people

### DISCUSSION THREAD

**Discussion by knutikov** [NightscoutFoundation/xDrip#3090] [Bug reports]
Hi, I was able to set the Xdrip+ watch face on my android wear, however since I'm left handed I configured my watch to be used on my right hand and rotated the screen. Everything looks normal but the all xDrip+ faces look upside down.

![20230918_190412](https://github.com/NightscoutFoundation/xDrip/assets/84673724/aa356e6d-573e-4d40-85c6-e7e84aeed9f2)
![20230918_190602](https://github.com/NightscoutFoundation/xDrip/assets/84673724/9176079f-2dc4-4330-9d6e-8a6c4218a735)

Thanks for your kind help.

**Comment by Navid200** [comment]
We don't have an active developer looking after WearOS unfortunately.
If there is anyone interested, they can open a PR.
